CMakeLists.txt

```diff
@@ -86,10 +86,6 @@ if(TRITON_ENABLE_TRACING AND NOT TRITON_ENABLE_STATS)
   message(FATAL_ERROR "TRITON_ENABLE_TRACING=ON requires TRITON_ENABLE_STATS=ON")
 endif()
 
-if (TRITON_ENABLE_METRICS_CPU AND NOT TRITON_ENABLE_METRICS)
-  message(FATAL_ERROR "TRITON_ENABLE_METRICS_CPU=ON requires TRITON_ENABLE_METRICS=ON")
-endif()
-
```

Comment:
FYI: Make sure to rebase so these Metrics additions don't get deleted

Reply:
Strange, I'm not sure how these were overwritten, I always make sure to rebase before a push. Regardless, they have been manually re-added.